src/skins/Crypt/Crypt.tsx

```diff
@@ -0,0 +1,250 @@
+import React, {PropsWithChildren, useState} from 'react';
+import './Crypt.scss';
+import {IProfileProps} from '../../models';
+import crypto from 'crypto';
+
+var key = 0
+
+function crypt(text?: string) {
+  let d1 = key ? key % 26 : 0
+  let d2 = key ? key / 26 % 26 : 0
+  return text?.split('').map((c, j) => {
+    let d = j % 2 === 0 ? d1 : d2
+    var i = c.charCodeAt(0)
+    if (i >= 65 && i <= 90) i = (i - 65 + d) % 26 + 65
+    else if (i >= 97 && i <= 122) i = (i - 97 + d) % 26 + 97
+    return String.fromCharCode(i)
+  }).join('')
+}
+
+export function Crypt(props: PropsWithChildren<IProfileProps>) {
+  const {basics, skills, work, education, references, projects, publications, languages} = props.profile;
+
+  const [value, setValue] = useState(0);
+  let update = (event: React.FormEvent<HTMLInputElement>) => {
+    const orig = event.currentTarget.value
+    const update = orig.split('').filter(c => c >= '0' && c <= '9').join('')
+    if (orig !== update)
+      event.currentTarget.value = update
+    const nv = parseInt(update)
+    if (!isNaN(nv))
+      setValue(value => nv)
+  }
+
+  key = ((crypto.createHash('sha256').update(basics?.name || "Anonymous").digest().readUInt16BE(0) - value) % 676 + 676 ) % 676
+
+  return (
+    <div className="basic-skin">
```

Comment:
Can you change the `className` to `crypt-skin` so we won't have duplicates? (same for the CSS).